src/Microsoft.Sbom.Parsers.Spdx22SbomParser/Validator.cs

```diff
@@ -14,17 +15,17 @@ public class Validator : IManifestInterface
     {
         public string Version { get; set; }
 
-        private readonly ManifestInfo spdxManifestInfo = new ManifestInfo
+        private readonly ManifestInfo spdxManifestInfo = new ()
         {
             Name = Constants.SPDXName,
             Version = Constants.SPDXVersion
         };
 
-        public ManifestData ParseManifest(string manifest)
-        {
-            throw new NotImplementedException($"Currently we don't support validating SPDX 2.2 SBOMs");
-        }
+        public ManifestData ParseManifest(string manifest) 
+            => throw new NotImplementedException($"Currently we don't support parsing complete SPDX 2.2 SBOMs");
```

Comment:
>Currently we don't support parsing complete SPDX 2.2 SBOMs" [](http://example.com/codeflow?start=51&length=59)

What does this mean?

Reply:
It means that we don't support parsing the whole SBOM file at once... I will update the string to be a little better